cookbook/request/load_balancer_reverse_proxy.rst

```diff
@@ -95,6 +98,27 @@ That's it! It's critical that you prevent traffic from all non-trusted sources.
 If you allow outside traffic, they could "spoof" their true IP address and
 other information.
 
+My Reverse Proxy sends X-Forwarded-For but does not filter the Forwarded header
+-------------------------------------------------------------------------------
+
+Many popular proxy implementations do not yet support the Forwarded header and
+does not filter it by default configuration. Ideally, you would configure this
+in your proxy, but if this is not possible, you can tell Symfony to distrust
```

Comment:
this is a long sentence, you might want to split it up here: "in your proxy. If this is not possible, you can [...]"